scripts/install-jdk.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+. ~/.jabba/jabba.sh
+
+# Defaults to Java 1.8
+jdk_to_install=${TRAVIS_JDK:-8}
+
+# Ensure the build will use the latest JDK for that major release
+jabba_identifier=$(jabba ls-remote adopt@ | grep "1.${jdk_to_install}" | sort -r | head -1)
+
+jabba install "$jabba_identifier" && jabba use "$_" && java -Xmx32m -version
```

Comment:
Will move to SDKMAN in the future. Let's stick to jabba.

Reply:
Huh, the jabba version is pretty simple too...

Reply:
Actually, it looks like there are issues in using jabba for AdoptOpenJDK versions: https://github.com/shyiko/jabba/issues/481 😕